ci/prow/boskos/set_permissions.sh

```diff
@@ -78,7 +83,6 @@ done
 echo "Activating GCS service account"
 curl -s -X GET -H "Authorization: Bearer ${ACCESS_TOKEN}" "https://www.googleapis.com/storage/v1/projects/${PROJECT}/serviceAccount"
 GCS_SERVICE_ACCOUNT="service-${PROJECT_NUMBER}@gs-project-accounts.iam.gserviceaccount.com"
```

Comment:
Then also remove this line?

Reply:
```suggestion
```

Reply:
I messed up my branch, this line was added back and I missed it. :P 
Removed, thanks!